content/maintainers-guide/how-to-become-a-committer.md

```diff
@@ -13,15 +13,9 @@ Committers participate in reviewing code contributions and are able to include t
 ## Requirements to become a committer
 
 1. **Active participation in the project**
-    - Display a record of several accepted, quality Pull Requests having received little or no objections from maintainers.
-    - Display a record of Pull Request reviews.
-    - Be active in the project’s [Slack][slack] chat.
+    - Have at least one accepted, quality Pull Request
```

Comment:
I think this part should be kept, otherwise we'll end up with useless committer applications, or people who will only contribute with useless "LGTM"s
```suggestion
    - Have at least one accepted, quality Pull Request.
    - Display a record of Pull Request reviews.
```